### PR TITLE
Improve hurt command feedback and usability

### PR DESCRIPTION
**Pull request by DrSmugleaf**

### CODE REVIEW

File: Content.Server/Commands/HurtCommand.cs [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+            {
+                msg += $"\n{dClass}";
+                var types = Enum.Parse<DamageClass>(dClass).ToTypes();
+
+                if (types.Count > 0)
+                {
+                    msg += ": ";
+                }
+
+                msg += string.Join('|', types);
```
Comment (metalgearsloth) [inline Content.Server/Commands/HurtCommand.cs:36]: StringBuilder

File: Content.Server/Commands/HurtCommand.cs [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+            switch (args.Length)
+            {
+                // Check if we have enough for the dmg types to show
+                case var n when n > 0 && (args[0] == "?" || args[0] == "¿"):
+                    var types = DamageTypes();
+
+                    if (args[0] == "¿")
+                    {
+                        types = types.Replace("e", "é");
+                    }
```
Comment (metalgearsloth) [inline Content.Server/Commands/HurtCommand.cs:177]: Spanish moment. Suss on this.

**Review by metalgearsloth — CHANGES_REQUESTED** [review]

File: Content.Server/Commands/HurtCommand.cs (hunk 1 above)
Comment (DrSmugleaf) [reply Content.Server/Commands/HurtCommand.cs:36]: Done.

File: Content.Server/Commands/HurtCommand.cs (hunk 2 above)
Comment (DrSmugleaf) [reply Content.Server/Commands/HurtCommand.cs]: I don't know what you méan sir

**Comment by DrSmugleaf** [comment]
Reviews addressed.

**Review by metalgearsloth — APPROVED** [review]

**Pull request merged**